samples/vsphere/appliances/health_messages.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+"""
+* *******************************************************
+* Copyright (c) VMware, Inc. 2019. All Rights Reserved.
+* SPDX-License-Identifier: MIT
+* *******************************************************
+*
+* DISCLAIMER. THIS PROGRAM IS PROVIDED TO YOU "AS IS" WITHOUT
+* WARRANTIES OR CONDITIONS OF ANY KIND, WHETHER ORAL OR WRITTEN,
+* EXPRESS OR IMPLIED. THE AUTHOR SPECIFICALLY DISCLAIMS ANY IMPLIED
+* WARRANTIES OR CONDITIONS OF MERCHANTABILITY, SATISFACTORY QUALITY,
+* NON-INFRINGEMENT AND FITNESS FOR A PARTICULAR PURPOSE.
+"""
+
+__author__ = 'VMware, Inc.'
+__copyright__ = 'Copyright 2019 VMware, Inc. All rights reserved.'
+__vcenter_version__ = '6.7+'
+
+from datetime import datetime, date
+from samples.vsphere.common import (sample_cli, sample_util, vapiconnect)
+from com.vmware.appliance_client import Health
+from com.vmware.vapi.std_client import LocalizableMessage
+from com.vmware.appliance_client import Notification
+
+
+class HealthMessages(object):
+    """
+    Demonstrates getting Health messages for various health items
+
+    such as memory, storage and cpu
+
+    Retrieves Health messages details from vCenter and prints the data
+
+    """
+
+    def __init__(self):
+        self.item = None
+        self.stub_config = None
+
+    def setup(self):
+        parser = sample_cli.build_arg_parser()
+
+        parser.add_argument(
+            '--item',
+            required=True,
+            action='store',
+            choices=['memory', 'cpu', 'storage'],
+            help='Specify the name of health item to view the messages')
+
+        args = sample_util.process_cli_args(parser.parse_args())
+        self.item = args.item
+
+        # Connect to vAPI services
+        self.stub_config = vapiconnect.connect(
```

Comment:
Can you use vsphere_client instead? https://github.com/vmware/vsphere-automation-sdk-python/blob/master/sample_template/sample_template_basic.py#L52